features/nanostack/mbed-mesh-api/mbed_lib.json

```diff
@@ -108,6 +108,22 @@
         "thread-security-policy": {
             "help": "Commissioning security policy bits [0-0xFF]",
             "value": 255
+        },
+        "ws-fhss-bc-interval": {
+            "help": "Broadcast interval. Duration between broadcast dwell intervals. Range: 0-16777216 milliseconds",
+            "value": 1000
+        },
+        "ws-fhss-bc-dwell-interval": {
+            "help": "Broadcast dwell interval. Range: 15-250 milliseconds",
+            "value": 200
+        },
+        "ws-fhss-uc-dwell-interval": {
+            "help": "Unicast dwell interval. Range: 15-250 milliseconds",
+            "value": 150
+        },
+        "ws-device-type": {
+            "help": "Device mode (NET_6LOWPAN_ROUTER or NET_6LOWPAN_HOST). Router is routing packets from other device, creating a mesh network.",
```

Comment:
I think we need domain, channel function and class configure I Guess. Jarkko please comment that If there is something what we not need and it based on platform